roadmap/cs-roadmap-2023.md

```diff
@@ -0,0 +1,93 @@
+  beyond the cost of the underlying Azure resources used by the user. It also helps to reduce costs
+  by allowing users to automate tasks and work more efficiently.
+
+Based on our tenets, these are the investment priorities for 2023.
+
+### Improved User Interface
+
+The user interface refresh will help improve the customer experience, accessibility, and
+functionality of Cloud Shell. This will make it easier for users to navigate and find the tools and
+resources they need. Benefits include:
+
+- Improved user interface to be more modern and intuitive.
+- Accessibility improvements for users who may have disabilities or impairments by improving color
+  contrast, adjusting font sizes and other accessibility improvements.
+- Support for enhanced functionality
+
+### Ephemeral Sessions
+
+Ephemeral sessions allow users to manage azure resources faster without the overhead of creating and
+associating Azure storage accounts. These temporary and disposable sessions are best for scenarios
+where customers need to perform ad-hoc tasks without the need to persist and data or files between
+sessions. In addition, this provides a layer of security to scenarios to ensure that data or files
+created during the session are not accessible to other users or processes after the session ends.
+
+Benefits include:
+
+- Allow customers to leverage Cloud Shell without a linked storage account
+- Any saved files will be deleted with the container session
+- Can be leveraged by partners to ensure that a customer can obtain a Cloud Shell session even if a
+  storage account has never been configured
+- Benefits command injection scenarios
+
+### Improving VNET Configuration
+
+Azure Cloud shell can be deployed to a local network (VNET) to gain access to local resources such
+as virtual machines, databases, and other connected services. As popularity of VNET has increased,
+so has a number of issues relating to configuration confusion and the need to improve both the
+documentation and configuration of VNET scenarios.
+
+The VNET refresh will address these issues by making improvements:
+
+- Updating documentation to better guide customers through the process of enabling a VNET.
+- Improving the configuration options of the VNET scenario.
+- Providing guided assistance to customers through ready-made ARM templates.
+
+## FY2024 Investments, Investigations and discussions
+
+Working with customers, we are investigating and discussing future areas of imporvmeent and new features.  These investigation are greatly impacted and enhanced with customer feedback. We welcome your feedback as we progress through these discussions by adding issues or discussions to our repository.
+
+- Multiple saved storage accounts
+- Multiple/Custom images
+- Improved editor experience
+- Tenent/Subscription Administrative controls to manage Cloud Shell access.
+- Toolbar tabs
+- Improvements to Cloud Shell timeout values
```

Comment:
What is this list? It seems disconnected from the previous paragraph.

```suggestion
- Multiple saved storage accounts
- Multiple/Custom images
- Improved editor experience
- Tenent/Subscription Administrative controls to manage Cloud Shell access.
- Toolbar tabs
- Improvements to Cloud Shell timeout values
```

Reply:
Agreed -- added context